### PR TITLE
tests: Relocate migration network test

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:

Move the migration network test from the network tests to the migration
tests.

The migration network option is maintained and focus on the compute area
and less in the network domain. In other words, it does not test network
components, but the migration ability to use a secondary network.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Review by enp0s3 — APPROVED** [review]
/lgtm
/approve

**Comment by EdDev** [comment]
/hold

something is wrong/missing.

**Comment by EdDev** [comment]
> something is wrong/missing.

The migration network test requires Multus, but that is only available in the sig-network jobs.

**Comment by EdDev** [comment]
@jean-edouard , could you please share your opinion on this one?

**Comment by jean-edouard** [comment]
Thanks for the heads-up!
The idea was to avoid installing CNAO on compute lanes, and restrict secondary network emulation to the quicker network lanes.
I don't mind this PR but maybe we should have a quick look at the time/resource overhead caused by it before merging?
Also, I think the secondary network could then be removed from the network lane, as it was added by me for this test.

**Comment by EdDev** [comment]
> The idea was to avoid installing CNAO on compute lanes, and restrict secondary network emulation to the quicker network lanes. I don't mind this PR but maybe we should have a quick look at the time/resource overhead caused by it before merging? 

The migration tests now work in a specialized jobs, so it should be fast.
CNAO deployment is cheap IMO, but I will try to measure it.

> Also, I think the secondary network could then be removed from the network lane, as it was added by me for this test.

Right, I will take it out. Thanks.

@AlonaKaplan , could you share your opinion here?

**Comment by EdDev** [comment]
@jean-edouard , the deployment of CNAO is occurring as part of `make cluster-up` and my local run test results shows that it takes them 10sec to be deployed.
Even if it is doubled on the CI, I think it is negligible.

**Comment by jean-edouard** [comment]
> @jean-edouard , the deployment of CNAO is occurring as part of `make cluster-up` and my local run test results shows that it takes them 10sec to be deployed. Even if it is doubled on the CI, I think it is negligible.

TBH I don't see the point of this...
If the test in question was completely unrelated to networking I'd understand (then it probably wouldn't need CNAO anyway!).
If there was no overhead to moving the test I'd be happy to let you move it pretty much wherever.
But in this case, because we decided that "migration network" is more migration than network, we're installing a whole bunch of add-ons to our busiest lane.
Sorry if this is a bit blunt, but am I missing something?

**Comment by EdDev** [comment]
> TBH I don't see the point of this... If the test in question was completely unrelated to networking I'd understand (then it probably wouldn't need CNAO anyway!). If there was no overhead to moving the test I'd be happy to let you move it pretty much wherever. But in this case, because we decided that "migration network" is more migration than network, we're installing a whole bunch of add-ons to our busiest lane. Sorry if this is a bit blunt, but am I missing something?

We splitted the e2e tests jobs into sigs to have focused owners.
I think this scenario that is checked here is not owned by the network sig, which makes it problematic to maintain.

I understand that we can have exceptions, but depending on CNAO is no less problematic than the compute sig depending on CDI.

**Comment by jean-edouard** [comment]
> > TBH I don't see the point of this... If the test in question was completely unrelated to networking I'd understand (then it probably wouldn't need CNAO anyway!). If there was no overhead to moving the test I'd be happy to let you move it pretty much wherever. But in this case, because we decided that "migration network" is more migration than network, we're installing a whole bunch of add-ons to our busiest lane. Sorry if this is a bit blunt, but am I missing something?
> 
> We splitted the e2e tests jobs into sigs to have focused owners. I think this scenario that is checked here is not owned by the network sig, which makes it problematic to maintain.
> 
> I understand that we can have exceptions, but depending on CNAO is no less problematic than the compute sig depending on CDI.

That last part is a good point!

@dhiller @rmohr any opinion on this?

**Pull request merged**
